tests/manage/mcg/test_bucket_creation.py

```diff
@@ -4,30 +4,54 @@
 
 from ocs_ci.framework import config
 from ocs_ci.framework.pytest_customization.marks import tier1
+from ocs_ci.utility.utils import check_if_executable_in_path
 
 logger = logging.getLogger(__name__)
 
 
-@pytest.mark.skipif(condition=True, reason="MCG is not deployed")
-@pytest.mark.filterwarnings('ignore::urllib3.exceptions.InsecureRequestWarning')
+@pytest.mark.filterwarnings(
+    'ignore::urllib3.exceptions.InsecureRequestWarning'
+)
+@pytest.mark.skipif(
+    condition=config.ENV_DATA['platform'] != 'AWS',
+    reason="Tests are not running on AWS deployed cluster"
+)
 @tier1
 class TestBucketCreation:
     """
     Test creation of a bucket
     """
-
-    @pytest.mark.skipif(
-        condition=config.ENV_DATA['platform'] != 'AWS',
-        reason="Tests are not running on AWS deployed cluster"
-    )
     @pytest.mark.polarion_id("OCS-1298")
     def test_s3_bucket_creation(self, mcg_obj, bucket_factory):
         """
         Test bucket creation using the S3 SDK
         """
-
         assert set(
-            bucket.name for bucket in bucket_factory(3)
+            bucket.name for bucket in bucket_factory(3, 'S3')
         ).issubset(
             mcg_obj.s3_list_all_bucket_names()
         )
+
+    @pytest.mark.skipif(
```

Comment:
Can you specify the skipif marker here: 

https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/framework/pytest_customization/marks.py#L55

with name nooba_cli_required or something like that? If you will have more such tests you can just reuse the same skipif marker imported from there.